pwndbg/commands/__init__.py

```diff
@@ -259,7 +259,7 @@ def _OnlyAmd64(*a, **kw):
 def OnlyWithLibcDebugSyms(function):
     @functools.wraps(function)
     def _OnlyWithLibcDebugSyms(*a, **kw):
-        if pwndbg.heap.current.libc_has_debug_syms():
+        if pwndbg.heap.current.libc_has_debug_syms() or pwndbg.config.resolve_via_heuristic:
             return function(*a, **kw)
         else:
             print('''%s: This command only works with libc debug symbols.
```

Comment:
We should improve the messaging here, but we can work it out on Discord after merging this

Reply:
Sure!
You mean the hint messaging for it, right?

Reply:
yeah